### ISSUE TITLE
are metrics (like tp_fp_fn_tn) still pending to port to 0.3?

### ISSUE THREAD

**Issue by jpchavat** [nilmtk/nilmtk#821]
I was inspecting what kind of metrics I can use to compare my algorithms and found that there are some of them commented in the code:

https://github.com/nilmtk/nilmtk/blob/a7051c069dba14e670a0daca92911e80ad1cf8db/nilmtk/metrics.py#L248

https://github.com/nilmtk/nilmtk/blob/a7051c069dba14e670a0daca92911e80ad1cf8db/nilmtk/metrics.py#L282-L283

My question is, are this metrics still pending of porting to version 0.3? or were it moved to another place in the code?

**Comment by PMeira** [comment]
Yes, seems to be missing still. Shouldn't be too hard to port most of those though.
